pkg/mfa/client.go

```diff
@@ -294,3 +304,80 @@ func (c *Client) VerifyChallenge(
 	}
 
 }
+
+// DeleteFactor deletes a Factor.
+func (c *Client) DeleteFactor(
+	ctx context.Context,
+	opts DeleteFactorOpts,
+) error {
+	c.once.Do(c.init)
+
+	endpoint := fmt.Sprintf(
+		"%s/auth/factors/%s",
+		c.Endpoint,
+		opts.ID,
+	)
+	req, err := http.NewRequest(
+		http.MethodDelete,
+		endpoint,
+		nil,
+	)
+	if err != nil {
+		return err
+	}
+
+	req = req.WithContext(ctx)
+	req.Header.Set("Authorization", "Bearer "+c.APIKey)
+	req.Header.Set("Content-Type", "application/json")
+	req.Header.Set("User-Agent", "workos-go/"+workos.Version)
+
+	res, err := c.HTTPClient.Do(req)
+	if err != nil {
+		return err
+	}
+	defer res.Body.Close()
+
+	return workos_errors.TryGetHTTPError(res)
+}
+
+// GetFactor gets a Factor.
```

Comment:
nit: change comment to `Retrieves an authentication factor` to be in line with other methods.